packages/clay-form/src/Checkbox.tsx

```diff
@@ -76,13 +81,13 @@ const ClayCheckbox = React.forwardRef<HTMLInputElement, IProps>(
 				<label>
 					<input
 						{...otherProps}
-						aria-disabled={readOnly}
+						aria-disabled={disabled}
```

Comment:
As we already have the `disabled` property I think we don't need `aria-disabled` here, it would only be good for the case where we don't have `disabled` and also when we want the component to have a certain interaction and be announced by SR .

Reply:
Thanks @matuzalemsteles!
Removed.